UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -836,6 +836,16 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
         /// </summary>
         /// <param name="format">The format string.</param>
         /// <returns>The string representation.</returns>
+        /// <remarks>
+        /// The valid format strings are as follows:
+        /// "g": The quantity formatted as the value with 2 significant digits after the radix, and abbreviation. For example: 1.23 m
+        /// "a": The default abbreviation for the quantity's unit.
+        /// "a0", "a1", "a2", etc.: The abbreviation with index where multiple abbreviations exist. "a0" is the same as "a" or the default abbreviation.
```

Comment:
`"a0", "a1", ..., "aN": The Nth unit abbreviation for <see cref="Unit" />. "a0" is the same as "a".`

What happens if you specify a value out of range? That should be documented too.